go-controller/pkg/ovn/egressip.go

```diff
@@ -2057,6 +2068,16 @@ func createLogicalRouterPolicy(nbClient libovsdbclient.Client, match string, pri
 	return nil
 }
 
+func deleteDefaultRerouteToExternal(nbClient libovsdbclient.Client, nodeName string) error {
+	p := func(lrsr *nbdb.LogicalRouterStaticRoute) bool {
+		return lrsr.ExternalIDs["egressip-on-ic-catch-all"] == nodeName
+	}
+	if err := libovsdbops.DeleteLogicalRouterStaticRoutesWithPredicate(nbClient, types.OVNClusterRouter, p); err != nil {
```

Comment:
I guess there is no issue having possibly two duplicated routes along with the kubevirt one.

I wonder if ovn controller consolidates this as a single route

Also worth noting for kubevirt we don't bother with deleting the route

Reply:
@tssurya after talking with @trozet, we think it would be better if this is the same exact route as the one we have for live migration instead of being duplicated. Because of that , we also shouldn't bother removing the route once it has been added, just as live migration doesn't.

Reply:
ack just catching up on this one.. let me check how HCP live-migration code does this and maybe we can just reuse that if its the exact same route match?

Reply:
so i have removed the deletion logic here for when the node is deleted based on this comment.